integration/mochaTests/README.md

```diff
@@ -0,0 +1,131 @@
+
+### Integration tests
+Integration tests are minimal tests that run on latest Chrome on MacOs. These tests are used to test the basic functionality across the more popular browsers.
+
+### Browser Compatibility Tests
+Browser compatibility tests are used to test a feature across several different browser and OS combinations. This test is used to make sure that JS SDK features are compatible with a wide variety of browser and OS. There is a default set of browser and OS that are used for this test. In a case where a feature is not compatible with one of the browser or OS then the user can pass in a custom config.
+
+In the following section, we will go over the schema of the custom JSON config.
+#### JSON Config Schema
+
+```json
+{
+   "clients": [
+     {
+        /* 
+          Browser name
+          @required
+        */ 
+        "browserName": "chrome",
+        /* 
+          Browser Version
+          @required
+        */
+        "browserVersion": "latest",
+        /* 
+          Platform
+          @required        
+        */
+        "platform": "MAC"
+     }
+   ]
+}
+```
+## Running tests
+
+### Running Integration Test Locally
+You can run the integration tests locally. By default, integration tests will run on latest version of Chrome that is installed on your machine.
+You will have to make sure that you have the required drivers installed on your machine that the selenium test uses. You can find more information about the different drivers available at the `selenium-webdriver` npm page: [https://www.npmjs.com/package/selenium-webdriver](https://www.npmjs.com/package/selenium-webdriver). 
+
+If you have an older version of driver installed then you will need to have an older version of browser on your machine as well. Generally, it is recommended to do local testing on latest version. If you need to test on older version, you can run a browser compatibility test with saucelabs.
+
+Sample command to run an integration test locally:
+
+```bash
+npm run test -- --test-name AudioTest --host local --test-type integration-test
+```
+
+Browser compatiblity tests will run across a variety of browser and OS combinations so it is recommended to use SauceLabs for them.
+
+### Running Browser Compatibility Tests on SauceLabs
+To run the test on SauceLabs, the username and access key will be required for authentication. Update the following commands with the credentials and add the credentials to environment variables.
+```bash
+export SAUCE_USERNAME=<Sauce Labs account username>
+```
+```bash
+export SAUCE_ACCESS_KEY=<Sauce Labs access key>
+```
+
+SauceLabs will open a browser and load a test url. You need to have a serverless deployment of Chime JS SDK demo. You can set the demo url as an environment variable with the following command:
+```bash
+export TEST_URL=<JS SDK Demo URL>
+```
+
+Following command can be used to run a browser compatibility tests with default settings on SauceLabs:
```

Comment:
- `Following command` => `The following command`
- `a browser compatibility tests` => `browser compatibility tests` (remove `a`)